pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -51,27 +51,27 @@ func (admitter *SecretAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissio
 	secret := &core.Secret{}
 	err := json.Unmarshal(raw, secret)
 	if err != nil {
-//		reviewResponse := admissionv1.AdmissionResponse{}
-//		reviewResponse.Allowed = false
-//		return &reviewResponse
+		//		reviewResponse := admissionv1.AdmissionResponse{}
+		//		reviewResponse.Allowed = false
+		//		return &reviewResponse
```

Comment:
This looks little bit weird now and could be reformatted or removed (what do you think @mnecas?) in a separate patch.